tests/test_event_updates.py

```diff
@@ -1,16 +1,67 @@
 """
 Test tap gets all updates for streams with updates published to the events stream
 """
-import json
-from time import sleep
-from random import random
-
-import requests
-from tap_tester import menagerie, runner, connections
+from datetime import datetime, timedelta
+import os
+from tap_tester import runner, connections
 from base import BaseTapTest
-from utils import \
-    get_catalogs, update_object, create_object, delete_object
+from utils import update_object, create_object, delete_object
+
+
+class TestEventUpdatesSyncStart(BaseTapTest):
+    """
+    Test for event update and event creation records of streams.
+    Even if the start date is set before 30 days, no record before 30 days will be received.
+    """
+    @staticmethod
+    def name():
+        return "tt_stripe_event_sync_start"
+
+    def get_properties(self, *args):
+        """Configuration properties required for the tap."""
+
+        props = super().get_properties(*args)
+        props['event_date_window_size'] = 35
```

Comment:
Add comment

Reply:
Added comment for `event_date_window_size`.